api/Window.json

```
@@ -2678,7 +2678,8 @@
           "spec_url": "https://html.spec.whatwg.org/multipage/indices.html#event-hashchange",
           "support": {
             "chrome": {
-              "version_added": "5"
+              "version_added": "7",
+              "notes": "Before Chrome 8, a regular <code>Event</code> instance was thrown, rather than a <code>HashChangeEvent</code> instance."
```

```suggestion
              "notes": "In Chrome 8, a regular <code>Event</code> object is fired, rather than a <code>HashChangeEvent</code> object."
```

But how about using two ranges for this, to hide away the irrelevant note for readers on MDN?